.pre-commit-config.yaml

```diff
@@ -0,0 +1,63 @@
+repos:
+  - repo: https://github.com/norwoodj/helm-docs
+    rev: v1.11.0
+    hooks:
+      - id: helm-docs-built
+        name: 'Helm Docs for Turing Chart'
+        files: '^infra/charts/turing/'
+        args:
+          - --chart-search-root=infra/charts/turing
+          - --template-files=./README.md.gotmpl
+      - id: helm-docs-built
+        name: 'Helm Docs for Turing Init Chart'
+        files: '^infra/charts/turing-init/'
+        args:
+          - --chart-search-root=infra/charts/turing-init
+          - --template-files=./README.md.gotmpl
+  - repo: https://github.com/golangci/golangci-lint
+    rev: v1.48.0
+    hooks:
+      # For the below hooks, we need to cd into the folder as we cannot execute golangci-lint
+      # outside the module. See: https://github.com/pre-commit/pre-commit/issues/1417
+      - id: golangci-lint
+        name: 'Linter for API'
+        files: '^api/'
+        types: [go]
+        entry: bash -c 'cd api && golangci-lint run --fix'
```

Comment:
Are we fine with the default timeout value now? It was previously set to 3 minutes

Reply:
It hasn't taken as long for me recently. But fair point, I added it back for the `api` module.